### ISSUE TITLE
Add temporary achievement progress

### ISSUE THREAD

**Issue by acenturyandabit** [acenturyandabit/fortknight#53]
Currently there are two ways of validating achievements: Either the achievement's progress is saved, or the achievement's progress is transient. For transient achievements, the progress bar doesn't show up. 

Edit the transient acheivements checking function and associated data so that a progress bar for the acheivements shows up.

**Comment by sudesh7447** [comment]
Hey I'm a beginner in open source but i think I can add this Enhancement. Can you assign this to me?

**Comment by PushpenderSaini0** [comment]
Yes @sudesh7447 , i am assigning you this, you can later submit a pull request mentioning this issue, i will review it and merge .
You can @ me whenever you need help !

**Comment by sudesh7447** [comment]
Thanks sir! can you explain the task a little bit more specifically?

**Comment by acenturyandabit** [comment]
Sure!
-------------------- Steps to reproduce -------------------
1. Open https://acenturyandabit.github.io/fortknight in an incognito window.
2. Click the "Scores and Achievements" tab. The first achievement should be "Lazy". 
3. Start a game. Click the knight to waste a move. You should see the "Lazy" bar fill up a bit.
4. Do another lazy move (Click the knight to waste a move). You should see the "Lazy" bar fill up a bit more.
5. Now, don't do any more lazy moves - keep moving and lose.

Intended behaviour: The "Lazy" bar keeps showing the 2 lazy moves that you did, in a different colour (maybe red).

Actual behavior: The "Lazy" bar clears, and you lose all your progress.
-------------------- Coding Hints --------------------------
Check out the file `js/achievements_highscores.js`.
